bigchaindb_driver/driver.py

```diff
@@ -63,11 +66,24 @@ def outputs(self):
         """
         return self._outputs
 
+    def node_info(self, headers=None):
+        return self.transport.forward_request(
+            method='GET', path='/', headers=None)
+
+    def api_info(self, headers=None):
+        return self.transport.forward_request(
+            method='GET',
+            path=self.api_prefix,
+            headers=None,
```

Comment:
The headers in the `forward_request()` for `node_info` and `api_info` should be `headers` instead of `None`?